ci/jenkins/test.sh

```diff
@@ -240,10 +240,12 @@ function collect_windows_network_info_and_logs {
 
         DOCKER_LOG_PATH="${DEBUG_LOG_PATH}/${NODENAME}/docker"
         mkdir "${DOCKER_LOG_PATH}"
-        scp -q -o StrictHostKeyChecking=no -T administrator@${IP}:'/cygdrive/c/"Program Files"/Docker/dockerd.log*' "${DOCKER_LOG_PATH}"
+        if [[ ${TESTCASE} != *"containerd"* ]];then
```

Comment:
This may work but could we stick to one expressison for consistency and reduce the possibility that error is introduced by new expression?
https://github.com/antrea-io/antrea/blob/c00309883b407a8d6b8c5138b241d27afdaa0800/ci/jenkins/test.sh#L1074

Reply:
Updated.